### DISCUSSION TITLE
Value is shared between different broswers

### DISCUSSION THREAD

**Discussion by am185760** [Blazored/LocalStorage#220] [Q&A]
for LocalStorage, when I am opening site in different browser like Edge or chrome, it is sharing values while getting.

About session storage..its value get null in new tab so I cant use this for login credentials

**Comment by chrissainty** [comment]
I'm not sure what you're asking here. This library just provides a C# interface to the local storage APIs. It doesn't control or define be behaviour of those APIs.
